examples/tutorials/Druggablity_assessment_with_Fpocket.ipynb

```diff
@@ -0,0 +1,2112 @@
+{
```

Comment:
<p>Rephrase suggestions:</p><ol><li>a lot of time --&gt; significant</li><li>make sure --&gt; ensure</li><li>responding to a drug --&gt; responding to the drug</li><li>due to the “undruggability” --&gt; , often due to the “undruggability”</li><li>in addition to other challenges. --&gt; among other challenges.</li><li>like fpocket --&gt; , like fpocket,</li></ol><p> </p><p>I noticed a few minor grammatical issues in the text that might benefit from a quick pass through a grammar checker. It might help enhance clarity and readability. </p><p> </p><p>PS: I am not great at grammar :) .Quillbot grammar check is pretty decent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4069/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>